tdrs-backend/tdpservice/settings/common.py

```diff
@@ -471,7 +471,11 @@ class Common(Configuration):
         'Email Admin Number of Access Requests' : {
             'task': 'tdpservice.email.tasks.email_admin_num_access_requests',
             'schedule': crontab(minute='0', hour='1', day_of_week='*', day_of_month='*', month_of_year='*'), # Every day at 1am UTC (9pm EST)
-        }
+        },
+        'Reindex elasticsearch documents': {
+            'task': 'tdpservice.search_indexes.tasks.reindex_elastic_documents',
+            'schedule': crontab(minute='0', hour='5', day_of_week='*', day_of_month='*', month_of_year='*'),  # Every day at 5am UTC (1am EST)
```

Comment:
every day is probably excessive and may become an availability issue. open to feedback on how often this should run

Reply:
What came out of the discussion with respect to re-indexing every deploy?

Reply:
Regardless of whether we re-index on deploy too, I think it would make sense for this cron to run on a Friday at midnight (or some obscure time). That way, if the re-index kills the app, or somethings else happens, it'll be less likely to effect users since it will be happening outside of (what im assuming) is their normal days/hours to be submitting data. That gives us/Alex a chance to fix the issue before a user would be expecting to submit data.